### PR TITLE
Fix postgres phrase search to respect language set in settings

### PR DESCRIPTION
**Pull request by dest81**
Currently Phrase search ignores language set in settings as opposite to PlainText search:
```
WAGTAILSEARCH_BACKENDS = {
    "default": {
        "BACKEND": "wagtail.search.backends.database",
        "SEARCH_CONFIG": "english",
    }
}
```

Plain search works as expected:
```
Page.objects.search("bla bla")
...
WHERE ... @@ (to_tsquery('english'::regconfig, '(''bla'' & ''bla'')'))
....
```
Phrase search doesn't:
```
Page.objects.search(Phrase("bla  bla"))
...
WHERE .... @@ (phraseto_tsquery('bla  bla'))   // note language is missing
...
```

### CODE REVIEW

**Comment by zerolab** [comment]
Thanks @dest81 
Any chance you could add a test or two for this?

**Comment by dest81** [comment]
> Thanks @dest81 Any chance you could add a test or two for this?

Hi @zerolab, I have added tests.

File: wagtail/search/tests/test_postgres_backend.py [hunk 1]
```diff
@@ -172,3 +173,55 @@ def test_index_without_upsert(self):
...
+        book = models.Book.objects.create(
+            title="Nu is beter dan nooit",
+            publication_date="1999-05-01",
+            number_of_pages=333,
+        )
+        self.backend.add(book)
+        self.book = book
+
+    def tearDown(self):
+        self.book.delete()
```
Comment (zerolab) [inline wagtail/search/tests/test_postgres_backend.py:205]: this should not be needed

File: wagtail/search/tests/test_postgres_stemming.py [hunk 2]
```diff
@@ -1,40 +0,0 @@
...
-        self.backend.add(ru_book)
-
-        results = self.backend.search("Голубое", models.Book)
-        self.assertEqual(list(results), [ru_book])
-
-        results = self.backend.search("Голубая", models.Book)
-        self.assertEqual(list(results), [ru_book])
-
-        results = self.backend.search("Голубой", models.Book)
-        self.assertEqual(list(results), [ru_book])
```
Comment (zerolab) [inline wagtail/search/tests/test_postgres_stemming.py:38]: question: why not use this as a basis in `test_postgres_backend.py`?

File: wagtail/search/tests/test_postgres_backend.py (hunk 1 above)
Comment (dest81) [reply wagtail/search/tests/test_postgres_backend.py]: Yes, you are right. Removed!

File: wagtail/search/tests/test_postgres_stemming.py (hunk 2 above)
Comment (dest81) [reply wagtail/search/tests/test_postgres_stemming.py:38]: 1. this test always was skipped for me probably due to wrong `backend_name` 
2. we want to test WAGTAILSEARCH_BACKENDS configuration, that's why changing language with SQL query is not very useful
3. I needed to test not only plain text search but also phrases. Basically new test does the same just in different language. For me it was easier to write test from scratch than rewriting bigger part of existing one.

Comment (zerolab) [reply wagtail/search/tests/test_postgres_stemming.py:38]: Fair points. Thank you

**Review by zerolab — APPROVED** [review]

**Comment by zerolab** [comment]
Merged in c550173

**Pull request closed without merging**